Dockerfile-test

```diff
@@ -4,7 +4,7 @@ RUN curl -sL https://deb.nodesource.com/setup_8.x | bash - \
 	&& apt-get install -y nodejs
 RUN npm install -g AugurProject/solium
 
-RUN wget --quiet --output-document /usr/local/bin/solc https://github.com/ethereum/solidity/releases/download/v0.4.13/solc-static-linux \
+RUN wget --quiet --output-document /usr/local/bin/solc https://github.com/ethereum/solidity/releases/download/v0.4.16/solc-static-linux \
```

Comment:
Looks like you upgraded to Solidity 0.4.16. How come you left it at 0.4.13 in the Solidity files?

Reply:
The upgrade to Solidity was done as part of troubleshooting, and it seemed reasonable to keep it since everything worked as is and the new version has some useful features.  I wanted to avoid touching any more files than necessary in this PR though, and bumping solidity version would touch _all_ .sol files.  I do plan on a follow-up PR that bumps us to Solidity 0.4.16 or later.